rrfs-test/validated_yamls/templates/obtype_config/abi_g18.yaml

```diff
@@ -83,6 +83,56 @@
 
   # Observation Filters (QC)
   # ------------------------
+       obs pre filters:
+       # Satellite Zenith Angle Check
+       - filter: Bounds Check
+         filter variables:
+         - name: brightnessTemperature
+           channels: *abi_g18_channels
+         test variables:
+         - name: MetaData/sensorZenithAngle
+         maxvalue: 65.
+         action:
+           name: reduce obs space
+
+       # from read_abi: Reject when cloud fraction is more than 30
+       - filter: Domain Check
+         filter variables:
+         - name: brightnessTemperature
+           channels: *abi_g18_channels
+         where:
+         - variable:
+             name: MetaData/cloudAmount
+             channels: 8
+           maxvalue: 30.
+         action:
+           name: reduce obs space
+
+       # from read_abi: WV Channels 8-10 scene consistency check SDTB>1.3
+       - filter: Domain Check
+         filter variables:
+         - name: brightnessTemperature
+           channels: *abi_g18_channels
+         where:
+         - variable:
+             name: ClearSkyStdDev/brightnessTemperature
+             channels: 8-10
+           maxvalue: 1.3
+           max_exclusive: true
+         action:
+           name: reduce obs space
+
+       # Data Thinning
+       - filter: Gaussian Thinning
+         #horizontal_mesh: 145
+         horizontal_mesh: 60
+         use_reduced_horizontal_grid: true
+         distance_norm: geodesic
+       #  round_horizontal_bin_count_to_nearest: true
+       #  partition_longitude_bins_using_mesh: true
+         actions:
+           - name: reduce obs space
+
```

Comment:
line 136:  FYI,
Per @HaidaoLin-NOAA's test, `obs prior filters` may not work as we expected and sometimes cause confusion.
We think it works if we put all `obs prior filters` as `obs post filters`.

Reply:
The obs prior filters for ABI only assign variables, which have worked very well in my entire baseline 1 retro experiment (4-days). 
The prior filters are different for different satellite. We can make another PR specifically to test  if we should move all prior filters to post filters. How do you think? @guoqing-noaa @HaidaoLin-NOAA

Reply:
Sure, that works for me. Thanks!